### ISSUE TITLE
BUG: to_latlon(4485585228861014020) yields NaN?

### ISSUE THREAD

**Issue by michaelleerilee** [SpatioTemporal/STARE#99]
4485585228861014020 -> nan in pystare

In hex: 0x3e40000000000004

**Comment by michaelleerilee** [comment]
Bug was not found on MacOS 11.5.1. The lon returned is `0`.

Verified the bug on AMCE Fire Daskhub where the lon returned is `NaN`.
```
(/home/jovyan/users_conda_envs/work) jovyan@jupyter-mrilee:~/git/STARE/build$ uname -a
Linux jupyter-mrilee 5.4.117-58.216.amzn2.x86_64 #1 SMP Tue May 11 20:50:07 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by michaelleerilee** [comment]
Verified bug on an Ubuntu 20 VM.

**Comment by michaelleerilee** [comment]
Fixed in 1.2.5.
